pkg/stack/stack.go

```diff
@@ -229,13 +240,18 @@ var dhis2CoreDefaults = struct {
 	imagePullPolicy:              ifNotPresent,
 	imageRepository:              "core",
 	imageTag:                     "2.40.2",
-	javaOpts:                     "",
+	javaOpts:                     " ",
```

Comment:
Is the added space intentional and if so why? It looks odd, so if intentional warrants a comment IMHO.